components/cards/EventWelcomeCard.jsx

```diff
@@ -0,0 +1,26 @@
+import classNames from "classnames/bind";
+import styles from "./cards.module.scss";
+import CTAButton from "../buttons/CTAButton";
+import Link from "next/link";
+
+const cx = classNames.bind(styles);
+
+const EventWelcomeCard = ({ event }) => {
+  //TODO: replace with dynamic props
+  return (
+    <div className={cx("event-card")}>
+      <p className={cx("title")}>Welcome to IAC22</p>
+      <p className={cx("description")}>
+        Some type of specific introduction and explanation on what the following
+        events are for. Example; Here you will find events on blank, they are
+        intended to do blank. something about social connections.
+      </p>
```

Comment:
You would be one step closer towards implementing the todo above defining a mock props object and using that to drive the placeholder UI.

Reply:
instead of hardcoding the text into the markup

Reply:
Totally agree! I'll be replacing this with a placeholder and apply that approach for future PRs.